### PR TITLE
Rescale transform

### PR DESCRIPTION
**Pull request by markvdw**
In situations where the gradients are very poorly scaled it can help to rescale the parameter space to make the gradients have a similar size to those of other parameters. This rescale transform can do this rescaling, while applying other transforms afterwards.

I think it definitely can be useful. Let me know if you agree and I'll implement some unittests.

### CODE REVIEW

**Comment by ialong** [comment]
I also need this quite badly!

**Comment by jameshensman** [comment]
Yep, sounds like a good feature.

File: GPflow/transforms.py [hunk 1]
```diff
@@ -195,6 +197,28 @@ def __str__(self):
         return '[' + str(self.a) + ', ' + str(self.b) + ']'
 
 
+class Rescale(Transform):
```
Comment (jameshensman) [inline GPflow/transforms.py]: Please add a docstring here.

**Comment by ialong** [comment]
Sweet! Thank you!

**Pull request merged**